conda_smithy/templates/run_docker_build.tmpl

```diff
@@ -26,9 +26,12 @@ show_channel_urls: true
 CONDARC
 )
 
+HOST_USER_ID=$( hash docker-machine 2>/dev/null && docker-machine ssh $(docker-machine active) 'id -u $(whoami)' || id -u $(whoami) )
```

Comment:
Ok, now I think I understand the question. I thought he was talking about running things inside the container as root, and not running docker as root.
So we really need to check if the id is 0 in the entrypoint